package.json

```diff
@@ -25,6 +25,7 @@
     "sdp-transform": "2.3.0",
     "strophe": "1.2.4",
     "strophejs-plugins": "0.0.7",
+    "webrtc-adapter": "webrtc/adapter#1eec19782b4058d186341263e7d049cea3e3290a",
```

Comment:
why this specific commit? is there no release we can use?

Reply:
webrtc-adapter didn't do a version bump after merging in a constraints fix that was desired for 1080p. I'm assuming it didn't happen because it'll be a major version bump so they're not rushing it.